README.md

```diff
@@ -0,0 +1,28 @@
+# GitHub Actions - Add PR to project
+
+Add community-created pull requests to the [Community Contributions project](https://github.com/orgs/silverstripe/projects/4) when they're opened.
+
+## Usage
+
+**.github/workflows/add-pr-to-project.yml**
+```yml
+name: Add pull request to project
+
+on:
+  pull_request:
+    types:
+      - opened
+
+permissions: {}
+
+jobs:
+  add-pr-to-project:
```

Comment:
```suggestion
  addprtoproject:
```
We seem to have a standard of no dashes in job names e.g. https://github.com/silverstripe/gha-pull-request

Reply:
Done.
https://github.com/silverstripe/gha-update-js has a hyphen though so I don't think we do have a standard. We just have a coincidence.